partcad/src/partcad/provider_data_cart.py

```
@@ -59,14 +55,18 @@ def __init__(self):
 
+    def _set_store_data(self, part):
+        store_data = part.get_store_data()
+        self.vendor = store_data.vendor
+        self.sku = store_data.sku
+        self.count_per_sku = store_data.count_per_sku
+
     async def set_spec(self, ctx, spec: str):
         self.name, self.count = resolve_cart_item(spec)
 
         part = ctx.get_part(self.name)
         assert part is not None
```

Ruff (0.8.2)

Use of `assert` detected

(S101)